test/components/length_test.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:test/test.dart';
+import 'package:turf/helpers.dart';
+import 'package:turf/length.dart';
+
+void main() {
+  test('length', () {
```

Comment:
Same here: It would be great to include an additional tests to verify if the default unit is set to kilometers and whether the unit parameter is used appropriately (rather than being hardcoded to meters).

Reply:
Good catch - added a such test.